ament_package/templates.py

```diff
@@ -15,13 +15,17 @@
 import os
 import re
 
-import pkg_resources
+try:
+    import importlib.resources as importlib_resources
+except ModuleNotFoundError:
+    import importlib_resources
```

Comment:
What is this fallback needed for?

Reply:
`importlib.resources` comes as part of the standard library in Python 3.8, which means we get it by default for Ubuntu 20.04, Windows, and macOS.  Ubuntu 18.04 can only get the module via a pip package (`importlib-resources`), so we need the fallback there.

Reply:
The doc states 3.7: https://docs.python.org/3/library/importlib.html#module-importlib.resources :thinking:

Reply:
Sure, I guess it came one release early.  I only have Ubuntu 18.04 (Python 3.6) and Ubuntu 20.04 (Python 3.8) handy, so I didn't test 3.7.

Reply:
Ok, so that it keeps working on (unsupported) Ubuntu 18.04. :+1: